doc/api/stream.md

```diff
@@ -325,6 +349,9 @@ implementations that implement the `writable.\_writev()` method can perform
 buffered writes in a more optimized manner.
 
 ##### writable.end([chunk][, encoding][, callback])
+<!-- YAML
+added: v0.9.9
+-->
```

Comment:
ok

On Thu, Jun 23, 2016 at 10:49 AM Anna Henningsen <notifications@github.com>
wrote:

> In doc/api/stream.md
> <https://github.com/nodejs/node/pull/7287#discussion_r68248685>:
>
> > @@ -325,6 +349,9 @@ implementations that implement the `writable.\_writev()` method can perform
> >  buffered writes in a more optimized manner.
> >
> >  ##### writable.end([chunk][, encoding][, callback])
> > +<!-- YAML
> > +added: v0.9.9
> > +-->
>
> @italoacasas <https://github.com/italoacasas> I’d still prefer to go with
> the version it was added in; adding some sort of changelog for individual
> methods is definitely planned.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7287/files/93d92a533fa6877b4902099ef13efebaa7501ecb#r68248685>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGD2F7ad126umPJeWbO7_aEbjCLZUfHbks5qOpzpgaJpZM4I0Y_b>
> .
>
-- 
Italo A. Casas